src/assets/stylesheets/styles.scss

```diff
@@ -100,17 +100,30 @@
 
 // 3. Load any custom SASS
 
+/* Custom utilities classes */
 // text styling
 .text-capitalize {
   text-transform: capitalize;
 }
 .text-break-all {
   word-break: break-all;
 }
+.text-ellipsis {
+  @include u-overflow('hidden');
+  @include u-text('no-wrap');
+  @include u-display(
+    'block'
+  ); // Use flex display on parent for inline elements.
+  text-overflow: clip;
```

Comment:
Non-blocking question: is there a reason to set both `clip` and `ellipsis`? (Like, is `clip` included here as a fallback?)

Reply:
`clip` is including as the fallback.

Reply:
....However, [looking at the spec](https://developer.mozilla.org/en-US/docs/Web/CSS/text-overflow#browser_compatibility), `ellipsis` is supported by the same number of devices and browsers and `clip`. I'll go ahead and remove `clip`.  Thanks for asking! Save a line of code.

Reply:
Nice! And same, thanks for talking through it with me!

Reply:
@beepdotgov Also, if you wrote this spec, ignore me.

![Homer disappearing into the bushes](https://i.giphy.com/media/v1.Y2lkPTc5MGI3NjExYzJhZXdxM2xlNzN1cTk2M2c0enlvOGVuaDFnYWl6eHViZGh1dXlvYyZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/a93jwI0wkWTQs/giphy.gif)

Reply:
@cannandev oh hey they don’t let me anywhere near specs, have you _seen_ me write css

![Homer Simpson sits in the middle of a room filled with impressive array of lights, buttons, and levers, spinning his chair around haphazardly.](https://github.com/user-attachments/assets/28d1f2a1-3947-4540-8eea-be87aa1cde37)